.github/workflows/tests.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         with:
           timeout_minutes: 5
           max_attempts: 5
-          command: composer require ramsey/collection:^1.2 brick/math:^0.9.3 --no-interaction --no-update
+          command: composer require ramsey/collection:^1.2 brick/math:^0.9.3 symfony/css-selector:^6.0 --no-interaction --no-update
```

Comment:
without this, `tijsverkoyen/css-to-inline-styles` allows requiring `symfony/css-selector` v2 with `--prefer-lowest`